tests/unit/api/search/test_v0.py

```diff
@@ -49,15 +49,15 @@ class SearchTableQueryString(unittest.TestCase):
     def setUp(self) -> None:
         self.mock_table_results = MOCK_TABLE_RESULTS
         self.expected_parsed_table_results = MOCK_PARSED_TABLE_RESULTS
-        self.search_url = local_app.config['SEARCHSERVICE_BASE'] + '/search_table'
+        self.search_url = local_app.config['SEARCHSERVICE_BASE'] + SEARCH_ENDPOINT
 
     def test_fail_if_term_is_none(self) -> None:
         """
         Test request failure if 'term' is not provided in the request json
         :return:
         """
         with local_app.test_client() as test:
-            response = test.post('/api/search/v0/table_qs', json={'pageIndex': 0})
+            response = test.post('/api/search/v0/table', json={'pageIndex': 0})
```

Comment:
Should `/table` be another constant?

Reply:
It could be part of the test setup but I don't see a need for it to be a defined constant. The only functional code that needs to know this endpoint is in the TypeScript code.